### ISSUE TITLE
Swift 5.8 isn't available on Windows

### ISSUE THREAD

**Issue by wadetregaskis** [swift-actions/setup-swift#560]
**Describe the bug**
"Version "5.8" is not available"

https://github.com/wadetregaskis/FoundationExtensions/actions/runs/4842921516/jobs/8630176019#step:2:5

**Workflow configuration (please complete the following information):**
 - Action version (`uses`): swift-actions/setup-swift@v1
 - Platform (`runs-on`): windows-latest
 - Swift version (`swift-version`): 5.8

**Comment by k-taro56** [comment]
This setup is not intended to work on Windows with Swift versions 5.7, 5.7.1, 5.7.2, 5.7.3, 5.8, 5.8.1, 5.9, 5.9.1, and 5.9.2. This is a specification, not a bug.
https://github.com/swift-actions/setup-swift/blob/cdbe0f7f4c77929b6580e71983e8606e55ffe7e4/src/swift-versions.ts#L6

**Comment by wadetregaskis** [comment]
Oh, that's disappointing.  Why has setup-swift dropped Windows support?

**Comment by k-taro56** [comment]
@wadetregaskis 
I agree with you. It seems that the reason is mentioned in the following link, but I would appreciate it if it was explicitly stated.
https://github.com/swift-actions/setup-swift/pull/499#issuecomment-1305864061
